config_test.ini

```diff
@@ -0,0 +1,17 @@
+[BDK]
```

Comment:
nit: this file is used in the tests. do you suggest to keep it separate from `config_example.json` ?

Reply:
I was going to keep it but it is a little confusing having it here. I found a simpler way to do my testing and was able to removed this file.